edx_adapt/tests/logic_tests.py

```diff
@@ -45,7 +45,9 @@ def _answer_pre_assessment_problems(self, correct=0, attention_question=True):
         """
         pre_assessments = ['Pre_assessment_{}'.format(i) for i in range(0, 14)]
         for i, problem in enumerate(pre_assessments):
```

Comment:
Correct parameter is described in doc strings but it could be changed.

i is nameless variable which in used for support role it is not clear how it should be named.